sdk/identity/azure-identity/README.md

```diff
@@ -46,7 +46,7 @@ To take dependency on a particular version of the library that isn't present in
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-identity</artifactId>
-    <version>1.14.2</version>
+    <version>1.15.0</version>
```

Comment:
Shouldn't this be `1.15.1`?

Reply:
It'll get fixed on the version-update. It gets weird since it is the dependency version, not the current version.